src/stylesheets/variables.scss

```diff
@@ -25,7 +25,7 @@ $button-shadow: 4px 4px 5px
 		$alpha: 0.25,
 	);
 
-$navbar-width: 90px;
+$navbar-width: 65px;
```

Comment:
If you end up changing things to vh, then this will need to be changed as well to make sure everything scales